feature-libs/product-configurator/rulebased/styles/_configurator-overview-sidebar.scss

```
@@ -0,0 +1,89 @@
+%cx-configurator-overview-sidebar {
+  &:not(:empty) {
+    display: flex;
+    flex-direction: column;
+    width: 100vw;
+    margin-inline-start: 25px;
+    padding-block-start: 5px;
```

@AlexanderKuschner: as the menu is on the left hand side and not on right hand side, the margin/padding shall be on the right hand side, as well. So it doesn't look odd. 
```suggestion
    margin-inline-end: 25px;
    padding-block-start: 5px;
```